### PR TITLE
Feat/styles

### PR DESCRIPTION
**Pull request by minjilee8816**

### CODE REVIEW

File: client/src/components/output.js [hunk 1]
```diff
@@ -101,57 +90,63 @@ class Output extends React.Component {
...
+          if ( items[j].itemName === itemAndPrice.itemName ) {
             items.splice(j, 1);
           } 
         }
       }
     }
   }
 
-  helperSetState (names) {
+  helperSetState () {
```
Comment (cosorio94) [inline client/src/components/output.js]: what happened to the names input variable?

File: client/src/components/output.js [hunk 2]
```diff
@@ -67,24 +69,11 @@ class Output extends React.Component {
     };
   }
 
-  friendInfo(name, number) {
```
Comment (cosorio94) [inline client/src/components/output.js]: What happened with friendInfo? Is this being used anywhere?

File: client/src/components/output.js [hunk 3]
```diff
@@ -101,57 +90,63 @@ class Output extends React.Component {
     };
 
     let debtors = this.state.debtors;
-    if (names.indexOf(name) === -1) {
-      this.addDebtor(debtor, itemAndPrice, names, foodList);
-    } else {
-      this.findDebtor(debtors, name, itemAndPrice, foodList);
+    if ( debtors.length === 0) {
+      this.addDebtor(debtor, itemAndPrice);
+    } else if ( debtors.length > 0 ) {
```
Comment (cosorio94) [inline client/src/components/output.js]: One of the changes that fixed a bug

File: client/src/components/output.js [hunk 4]
```diff
@@ -101,57 +90,63 @@ class Output extends React.Component {
...
+    for ( let i = 0; i < debtors.length; i++ ) {
+      if ( names.indexOf(debtors[i].name) === -1 ) {
         names.push(debtors[i].name); 
       }
     }
   }
 
   splitTax(debtorTotal) {
-    let percent = debtorTotal / this.props.total;
+    let percent = debtorTotal / (this.props.total - this.props.tax - this.props.tip);
```
Comment (cosorio94) [inline client/src/components/output.js]: Bug fix change

File: client/src/components/output.js [hunk 5]
```diff
@@ -101,57 +90,63 @@ class Output extends React.Component {
...
     let debtorTax = this.props.tax * percent;
     debtorTax = debtorTax.toFixed(2);
+    console.log('debtorTax: ', debtorTax);
     return Number(debtorTax);
   }
 
   splitTip(debtorTotal) {
-    let percent = debtorTotal / this.props.total;
+    let percent = debtorTotal / (this.props.total - this.props.tax - this.props.tip);
```
Comment (cosorio94) [inline client/src/components/output.js]: Bug fix change

File: client/src/components/output.js [hunk 6]
```diff
@@ -169,6 +164,7 @@ class Output extends React.Component {
       debtors[i]['debtTotal'] = debtTotal[i];
       debtors[i]['tax'] = this.splitTax(debtors[i].debtTotal);
       debtors[i]['tip'] = this.splitTip(debtors[i].debtTotal);
+      debtors[i]['debtTotal'] = debtTotal[i] + debtors[i]['tax'] + debtors[i]['tip'];
```
Comment (cosorio94) [inline client/src/components/output.js]: Bug fix change

File: client/src/components/output.js (hunk 1 above)
Comment (minjilee8816) [inline client/src/components/output.js]: I need to refactor the line of code. I put the names array in global variable in the way I didn't need to pass in any parameter. but when I refactor it then I will need to pass in the variable to helperSetState method.

File: client/src/components/output.js (hunk 2 above)
Comment (minjilee8816) [inline client/src/components/output.js]: We saved the friendInfo state in Redux so no need to have the friendInfo method anymore

**Pull request merged**